api/add.go

```diff
@@ -101,9 +102,14 @@ func AddParamsFromQuery(query url.Values) (*AddParams, error) {
 	params.Layout = layout
 
 	chunker := query.Get("chunker")
-	params.Chunker = chunker
+	if chunker != "" {
```

Comment:
why?

Reply:
default chunker is size-262144, which at this point is already set. This is avoiding setting it to empty.